hydra/_internal/utils.py

```diff
@@ -504,19 +504,36 @@ def _get_kwargs(config: Union[ObjectConf, DictConfig], **kwargs: Any) -> Any:
     return final_kwargs
 
 
+def _has_field(config: Union[ObjectConf, DictConfig], field: str) -> bool:
```

Comment:
It's hard to review when you mix your changes with refactoring like moving this function as is.
Try to separate cleanup to a different commit labeled as such to make the review easier.

Reply:
My bad. Will be more careful next time.